### DISCUSSION TITLE
Particle density dip at 1D boundary

### DISCUSSION THREAD

**Discussion by budjensen** [BLAST-WarpX/warpx#4988] [Q&A] [answered]
In the image below (from the capacitive discharge example of the WarpX documentation), the particle density dips at the boundary when compared to the benchmark result. Why is this? Is this related to the perfect electrical conductor boundary condition?

![image](https://github.com/ECP-WarpX/WarpX/assets/127121969/9fc09b45-08ea-4c1e-b649-8ee9341088fe)

**Comment by roelof-groenewald** [comment, marked as answer]
Hi @budjensen. Yes, this is due to the PEC boundary condition. In WarpX with a PEC the charge density deposited outside the simulation domain is reflected back into the domain with a sign flip (to mimic the effect of an image charge inside the conductor). See #3711.
